source/content/guides/sendgrid.md

```diff
@@ -71,6 +71,8 @@ The SendGrid API Integration Module for Drupal 8 requires a Composer-managed wor
 
 Your Drupal application on Pantheon is now set up to send email through SendGrid's API. Test your configuration from `/admin/config/services/sendgrid/test`.
 
+1. Set the system-wide default `MailSystemInterface` class. Select **SendGridMailSystem** from the dropdown menu. 
```

Comment:
@rachelwhitton Does this look okay?

Reply:
This new line should probably be moved up a few lines, so that it's above that final message saying "Your Drupal application on Pantheon is now set up.." 

Also, we should give them context for where this configuration option is, since it's not where we told them to go in step 1 

Suggestion: 

1. Go to `/admin/config/system/mailsystem` and set the system-wide default `MailSystemInterface` class to **SendGridMailSystem** from the dropdown menu.